examples/basic_usage/backend.tf

```
@@ -0,0 +1,10 @@
+terraform {
+  backend "s3" {
+    bucket         = "cisa-cool-terraform-state"
```

Same suggestion... make this match the tfvar example:

```suggestion
    bucket         = "example-terraform-state-bucket"
```

---

See comment above.  I'm going to keep the bucket name as is, unless you strongly disagree with that plan.